doc/src/vtr/run_vtr_flow.rst

```diff
@@ -173,3 +173,13 @@ Detailed Command-line Options
     Tells ODIN II to connect the first cin in an adder/subtractor chain
     to a global gnd/vdd net. Instead of creating a dummy adder to generate
     the input signal of the first cin port of the chain.
+
+.. option:: -odin_xml <path_to_custom_xml>
+
+    Tells VTR flow to use a custom ODIN II configuration value. The default
+    behavior is to use the vtr_flow/misc/basic_odin_config_split.xml. 
+    Instead, an alternative config file might be supplied, compare the 
```

Comment:
minor nit, should be:
"supplied; compare the"  (semicolon needed)

Reply:
Thank you @vaughnbetz  for the review. I have updated the code with the changes you suggested. Any other change you'd like me to implement or is the PR good to go?